spec/tags/subscriptions.md

```diff
@@ -24,6 +24,7 @@ The <code>code</code> field may assume the following values:
 * **DELIVERED**: The message sent by your business was delivered to the user's device.
 * **NOT_DELIVERED**: The message sent by your business failed to be sent to the user's device. The reason for the failure will be included in the <code>causes</code> field.
 * **READ**: The message sent by your business was read by the user. This status might not be sent, since it depends on the channel and on the user's device configuration.
+* **DELETED**: The message sent by the user was deleted by the user.
```

Comment:
Na linha 13 do arquivo, diz que os status são relacionados a mensagens enviadas anteriormente e agora estamos adicionando um status de mensagem recebida anteriormente. Acho que seria legal ajustar na linha 13 para deixar claro que os status podem ser para mensagens recebidas também.

Aproveitando, faz sentido adicionar algum indicativo de direção da mensagem nos status também? Como antes os status sera somente para mensagens OUT e agora estamos adicionando para IN também não sei se seria legal ter algo nesse sentido...

Reply:
Wlad, acredito que não seja necessário, pois este é um status enviado do provider para o cliente, assim como os demais. O propósito do filtro de mensagens acho que seria evitar o recebimento de mensagens que o cliente "já tem o conhecimento" (loopback) pq ele mesmo enviou. Neste caso é um status que o cliente não tem o conhecimento, ele não gerou.

Talvez faça sentido se tivermos status gerados pelo cliente, caso do READ para MO.

Reply:
Eu também pensei nisso, mas também considerando que o status é sempre referente a uma mensagem trafegada anteriormente, e que o cliente vai ter que casar com uma mensagem da base dele, acho que não é necessário, pois ele já deveria ter essa informação.